lib/ansible/modules/windows/win_domain_controller.ps1

```diff
@@ -257,7 +258,7 @@ Try {
             $local_admin_secure = $local_admin_password | ConvertTo-SecureString -AsPlainText -Force
 
             Write-DebugLog "Uninstalling domain controller..."
-            Uninstall-ADDSDomainController -NoRebootOnCompletion -LocalAdministratorPassword $local_admin_secure -Credential $domain_admin_cred
+            Uninstall-ADDSDomainController -NoRebootOnCompletion -LocalAdministratorPassword $local_admin_secure -Credential $domain_admin_cred -LogPath $log_path
```

Comment:
will this work? log_path is not necessarily defined. Shouldn't it be something like:
```suggestion
            Uninstall-ADDSDomainController -NoRebootOnCompletion -LocalAdministratorPassword $local_admin_secure -Credential $domain_admin_cred -LogPath:$log_path
```

Reply:
that parameter isn't relevant for this cmdlet apparently